pipeline/run_stability_tests.sh

```diff
@@ -74,7 +74,7 @@ cd report || exit
 grep -h "done at" ./*.txt | sort > dones &
 for asn in *.asn1; do
     asntool -m ../../lib_builder/asn.all \
-        -t Seq-annot -d "$asn" -p "$asn.txt" &
+        -t Seq-annot -d "$asn" -p "$asn.txt"
```

Comment:
@vartanianmh  have you considered GNU parallel? It works like a charm :)

Reply:
Isn't installed on Dataproc master and didn't seem worth fetching it.

Reply:
I think this can be done in another pull request.